src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/RoutingResolverPass.php

```diff
@@ -30,6 +30,12 @@ public function process(ContainerBuilder $container)
 
         $definition = $container->getDefinition('routing.resolver');
 
+        // routing.loader.directory only appears in Routing 2.7
```

Comment:
I would rather bump the requirement, so that people using FrameworkBundle 2.7 can be sure about which loaders are supported